src/src/pages/capabilities/SelectedCapabilityContext.js

```diff
@@ -109,6 +111,19 @@ function SelectedCapabilityProvider({ children }) {
     kafkaClusterTopicList();
   }, [clustersList]);
 
+  useEffect(() => {
+    if (members.length > 0) {
```

Comment:
Not sure this is necessary, assuming members are not undefined.
If they are, then this will not work either 🤷 

Not blocking

Reply:
That's fair ^^